benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java

```diff
@@ -380,6 +392,7 @@ private QueryExecutionResult executeSingleQuery(
             Optional<Path> outputFile)
             throws TimeLimitException
     {
+        LOG.info("Execute query, query=%s, skipReport=%s".formatted(benchmark.getQueries().get(0).getName(), Boolean.toString(skipReport)));
```

Comment:
Do we actually need that?

Reply:
> Do we actually need that?

It actually is nice when you can see in logs what query is executed in warmup phase and in normal measurement phase (skipReport=false)

Reply:
I don't think you need to call `Boolean.toString(`